src/ansys/templates/python/pyace_fastapi/hooks/post_gen_project.py

```diff
@@ -92,8 +89,10 @@ def main():
     # Remove docker non desired files
     enable_docker = "{{ cookiecutter.enable_docker }}"
     if enable_docker == 'Yes':
-        DESIRED_STRUCTURE.append("docker/Dockerfile")
-        DESIRED_STRUCTURE.append("docker/README.md")
+        DESIRED_STRUCTURE.append("docker-compose.yml")
+        DESIRED_STRUCTURE.append("Dockerfile")
+        DESIRED_STRUCTURE.append("Docker.md")
+        DESIRED_STRUCTURE.append(".dockerignore")
 
```

Comment:
I am already seeing something that is not a really good practice when dealing with Docker images. Why have you removed the `docker` directory from all templates? All Docker-related files should be contained inside this folder in order to avoid having even more files at the root level. If there is a reason for this, can you please share it with us @RVR06? Otherwise I would encourage to revert this. It is not a good practice.

Reply:
I support this. I'm 100% in favor of having all docker files under a ./docker directory.